src/main/perf/LineFileDocs.java

```diff
@@ -528,6 +532,7 @@ public Document nextDoc(DocState doc) throws IOException {
       int bodyLenBytes = buffer.getInt();
       int randomLabelLenBytes = buffer.getInt();
       timeSec  = buffer.getInt();
+      myID = buffer.getInt() + lfd.idOrBase;
```

Comment:
Hmm isn't `buffer.getInt()` an absolute `id`?  But then we add `lfd.idOrBase` and double-count the base?

Reply:
See comment below